### PR TITLE
template strings are adapted now

### PR DESCRIPTION
**Pull request by stevemao**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 No more string concatenation or array join!
 
-*Note that ES6 will have [template string](http://www.2ality.com/2011/09/quasi-literals.html) which can be multiline, but time...*
+*Note that it is recommended to use ES2015 [template strings](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/template_strings) if possible.*
```
Comment (kevva) [inline readme.md]: Maybe `it is` => `it's`?

Comment (sindresorhus) [inline readme.md]: I would go with something shorter:

> Use ES2015 template literals instead whenever possible.

Comment (kevva) [inline readme.md]: :+1:

**Comment by stevemao** [comment]
fixed :)

**Pull request merged**